atom/browser/browser_linux.cc

```diff
@@ -142,4 +142,39 @@ bool Browser::IsUnityRunning() {
   return unity::IsRunning();
 }
 
+void Browser::ShowAboutPanel() {
+  std::string app_name, version, copyright, icon_path, website;
+
+  GtkWidget* dialog = gtk_about_dialog_new();
+
+  if (about_panel_options_.GetString("applicationName", &app_name))
+    gtk_about_dialog_set_program_name(GTK_ABOUT_DIALOG(dialog),
+                                      app_name.c_str());
+  if (about_panel_options_.GetString("applicationVersion", &version))
+    gtk_about_dialog_set_version(GTK_ABOUT_DIALOG(dialog), version.c_str());
+  if (about_panel_options_.GetString("copyright", &copyright))
+    gtk_about_dialog_set_copyright(GTK_ABOUT_DIALOG(dialog), copyright.c_str());
+  if (about_panel_options_.GetString("website", &website))
+    gtk_about_dialog_set_website(GTK_ABOUT_DIALOG(dialog), website.c_str());
+  if (about_panel_options_.GetString("iconPath", &icon_path)) {
+    GdkPixbuf* icon = gdk_pixbuf_new_from_file(icon_path.c_str(), NULL);
+    gtk_about_dialog_set_logo(GTK_ABOUT_DIALOG(dialog), icon);
+    g_object_unref(icon), icon = NULL;
+  }
+
+  gtk_dialog_run(GTK_DIALOG(dialog));
+  gtk_widget_destroy(dialog);
```

Comment:
_(nit)_ `gtk_widget_destroy()` should be kept back for hardcore needs and also we leave a dangling pointer here.

More appropriate here would be `g_object_clear(&dialog);``